redash/models.py

```diff
@@ -658,10 +660,9 @@ def all_queries(cls, groups, drafts=False):
             .order_by(cls.created_at.desc())
 
         if drafts:
-            q = q.where(Query.name == 'New Query')
+            q = q.where(Query.is_draft == True)
         else:
-            q = q.where(Query.name != 'New Query')
-
+            q = q.where(Query.is_draft == False)
```

Comment:
So now all the "New Query" queries will appear every where before they didn't... Not sure how I feel about this. We could change all the existing ones to drafts, but then it will be different behavior than the one going forward :\